scripts/run-clang-format.sh

```diff
@@ -41,15 +48,23 @@ for e in ${EXCLUDE_DIRS[*]}; do
 done
 
 cd $SOURCE_DIR
-find . $FIND_EXCLUDES -type f \( $FIND_NAMES \) \
-  -exec clang-format -i -style=file \{\} +
+FILES=$(find . $FIND_EXCLUDES -type f \( $FIND_NAMES \) -print)
```

Comment:
Why do you need `-print` option here?

Reply:
When I do a "find . -path ./DPPP -prune -o -name *.cc" inside the DP3 directory, the output includes the ./DPPP directory, without any filename. When I add "-print", the output only includes the found *.cc files, and not that pruned directory name.

Reply:
Even with "-type f", the output still has the pruned directory name if I don't add "-print".